src/ui-test/baseExtensionTest.ts

```diff
@@ -0,0 +1,40 @@
+import { ExtensionsViewSection, ActivityBar, ExtensionsViewItem, QuickOpenBox } from 'vscode-extension-tester';
+import { ProjectInitializer } from './common/projectInitializerConstants';
+import { openCommandPrompt } from './common/commonUtils';
+import { expect } from 'chai';
+
+export function baseExtensionUITest() {
+    describe('Verify extension\'s base assets available after install', () => {
+
+        let inputBox: QuickOpenBox;
+
+        it('Command Palette prompt knows project initializer commands', async function () {
+            this.timeout(4000);
+            inputBox = await openCommandPrompt();
+            await verifyCommandPalette(inputBox);
+        });
+
+        it('Project initializer extension is installed', async function () {
+            this.timeout(5000);
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?